t/t.go

```diff
@@ -800,8 +800,10 @@ func downloadDataFiles() {
 		cmd.Dir = *tmp
 
 		if out, err := cmd.CombinedOutput(); err != nil {
-			fmt.Printf("Error %v", err)
-			fmt.Printf("Output %v", out)
+			fmt.Printf("Error %v /n", err)
+			str := string(out)
+			log.Println("error downloading a file: ", str)
+			log.Panic("error downloading required load test files")
```

Comment:
why don't we just do `panic("error downloading required load test files")` here instead?